packages/docs/getting-started/running-a-validator.md

```diff
@@ -512,7 +515,108 @@ You can now run the node for the attestation service in the background:
 docker run --name celo-attestations --restart always -p 8545:8545 -v $PWD:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --syncmode full --rpc --rpcaddr 0.0.0.0 --rpcapi eth,net,web3,debug,admin --unlock $CELO_ATTESTATION_SIGNER_ADDRESS
 ```
 
-By now, you should have setup your Validator account appropriately. You can finish the actual deploy of the attestation service under the [Attestation Service at the documentation page](running-attestation-service.md).
+A part of that we are going to setup the following environment variable about the Attestation Service Docker image:
```

Comment:
typos -- maybe `Next we will set up the Attestation Service itself. First, specify the following environment variables:`

also, could use some explanation that the "node" is different from the Attestation Service....